src/Xamarin.Android.Build.Tasks/Xamarin.Android.CSharp.targets

```diff
@@ -29,6 +29,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
         <TargetFrameworkIdentifier>MonoAndroid</TargetFrameworkIdentifier>
         <TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == ''">v5.0</TargetFrameworkVersion>
         <LangVersion Condition=" '$(LangVersion)' == '' ">8.0</LangVersion>
+        <AndroidAddKeepAlives Condition="'$(AndroidAddKeepAlives)' == ''">True</AndroidAddKeepAlives>
```

Comment:
Do we really want to do this for all `Debug` builds? What is the build time impact? I'm thinking of how many methods would get changed in support library or AndroidX assemblies.

Maybe we can set `$(AndroidAddKeepAlives)` to `true` when `$(AndroidIncludeDebugSymbols)` is `false`? Then it would be `Release` builds only by default.

Reply:
It is very rare to happen, so I think it should be fine to disable for Debug. That leads to producing different code for Debug and Release, hopefully the XF test we run on emulator should be enough to test that, for regressions, during our CI builds.